askup/utils/general.py

```diff
@@ -212,6 +212,7 @@ def get_user_correct_answers_count(user_id):
     """
     Return total amount of the correct answers of the user by id.
     """
+#    correct_answers = Answer.objects.aggregate(Count(id)).filter(self_evaluation=2, user_id=user_id).first()
```

Comment:
Remove commented line if it is not used

Reply:
Removing. It was a refactoring, I've started before.

Reply:
Done